### PR TITLE
Salvage cloak

### PR DESCRIPTION
**Pull request by lleftTheDragon**
## About the PR
Gave the Salvage cloak back to salvage, it rarely spawns in the locker and also should spawn in the salvage crates on salvages.

## Why / Balance
Added this awhile back, same reason as then. Wanted to give salvage their own cloak outside of the mining cloak as salvage isn't just about mining
 

## Media
Screenshot_798
Screenshot_799
Screenshot_800
Screenshot_801

- [ x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- Added Salvage cloak

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml [hunk 1]
```diff
@@ -115,6 +115,15 @@
...
     sprite: Clothing/Neck/Cloaks/trans.rsi
 
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckSalvager
+  name: salvager's cloak
+  description: Worn by dangerous people who've deconstructed stations with ease.
+  components:
+  - type: Sprite
+    sprite: Clothing/Neck/Cloaks/salvage.rsi
```
Comment (Colin-Tel) [inline Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml]: This should be under `DeltaV/Clothing/...`

File: Resources/Textures/Clothing/Neck/Cloaks/salvage.rsi/equipped-NECK3.png
Comment (Colin-Tel) [inline Resources/Textures/Clothing/Neck/Cloaks/salvage.rsi/equipped-NECK3.png:1]: What's the purpose of the `3` in the filename? If it needs to be there, then it needs to be present when you reference this file, hence the errors.

**Review by Colin-Tel — CHANGES_REQUESTED** [review]
Move the `salvage.rsi` folder into the same directory under `Textures/DeltaV/...`, also fix the errors in regards to the incorrect filename `equipped-NECK3.png`

File: Resources/Textures/Clothing/Neck/Cloaks/salvage.rsi/equipped-NECK3.png
Comment (lleftTheDragon) [reply Resources/Textures/Clothing/Neck/Cloaks/salvage.rsi/equipped-NECK3.png:1]: Fixed it, it was a hold over my B

File: Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml
Comment (Colin-Tel) [inline Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml:1]: Reverting any changes to this file will resolve some of the errors, maybe all of them.

Comment (lleftTheDragon) [reply Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml:1]: I Directly remember removing this code, i guess it didnt stick or i removed the change before comitting it? Either way fixed it


> Reverting any changes to this file will resolve some of the errors, maybe all of them.

File: Resources/Prototypes/Catalog/Fills/Crates/salvage.yml [hunk 2]
```diff
@@ -21,6 +21,8 @@
       - id: ClothingBeltUtility
       - id: OreBag
       - id: ClothingBeltSalvageWebbing
+      - id: ClothingNeckSalvager
```
Comment (Colin-Tel) [inline Resources/Prototypes/Catalog/Fills/Crates/salvage.yml]:
```suggestion
      - id: ClothingNeckSalvager # DeltaV - salvage cloak
```

File: Resources/Prototypes/Catalog/Fills/Lockers/cargo.yml [hunk 3]
```diff
@@ -25,6 +25,8 @@
         prob: 0.3
       - id: Flare
         prob: 0.3
+      - id: ClothingNeckSalvager
```
Comment (Colin-Tel) [inline Resources/Prototypes/Catalog/Fills/Lockers/cargo.yml]:
```suggestion
      - id: ClothingNeckSalvager # DeltaV - salvage cloak
```

File: Resources/Prototypes/Catalog/Fills/Lockers/cargo.yml [hunk 4]
```diff
@@ -48,3 +50,5 @@
...
```
Comment (Colin-Tel) [inline Resources/Prototypes/Catalog/Fills/Lockers/cargo.yml]:
```suggestion
      - id: ClothingNeckSalvager # DeltaV - salvage cloak
```

**Pull request merged**